### PR TITLE
docs: Add security policy file

### PR DESCRIPTION
**Pull request by validcube**
This is taken from https://github.com/ReVanced/revanced-manager/blob/dev/SECURITY.md

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+# 🔒 Security Policy
+
+This document describes how to report security vulnerabilities for ReVanced Manager.
+
+## 🚨 Reporting a Vulnerability
+
+Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
+
+If a vulnerability is confirmed and accepted, you can join our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
```
Comment (oSumAtrIX) [inline SECURITY.md:69]: How about instead reserving a security@revanced.app mail? This way you are not bound by service

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+     </picture>
+   </a>
+   <br>
+   <br>
+   Continuing the legacy of Vanced
+</p>
+
+# 🔒 Security Policy
+
+This document describes how to report security vulnerabilities for ReVanced Manager.
```
Comment (oSumAtrIX) [inline SECURITY.md:63]: Needs to be updated

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
Some references to ReVanced Manager have to be changed to ReVanced API

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
+
+If a vulnerability is confirmed and accepted, you can join our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
+
+### ⏳ Supported Versions
+
+| Version                                                                                                  | Branch      | Supported          |
+| -------------------------------------------------------------------------------------------------------- | ----------- | ------------------ |
+| ![Static Badge](https://img.shields.io/badge/version-latest-brightgreen?style=for-the-badge)             | dev         | :white_check_mark: |
```
Comment (oSumAtrIX) [inline SECURITY.md:75]: Maybe the main branch should be added as well

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+   Continuing the legacy of Vanced
+</p>
+
+# 🔒 Security Policy
+
+This document describes how to report security vulnerabilities for ReVanced Manager.
+
+## 🚨 Reporting a Vulnerability
+
+Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
```
[the remark this answers is not in the document]
Comment (validcube) [reply SECURITY.md:67]:
```suggestion
Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-api/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
```

File: SECURITY.md (hunk 2 above)
Comment (validcube) [reply SECURITY.md:63]:
```suggestion
This document describes how to report security vulnerabilities for ReVanced API.
```

File: SECURITY.md (hunk 3 above)
Comment (validcube) [reply SECURITY.md:75]:
```suggestion
| ![Static Badge](https://img.shields.io/badge/version-latest-brightgreen?style=for-the-badge)             | dev         | :white_check_mark: |
| ![Static Badge](https://img.shields.io/badge/version-latest-brightgreen?style=for-the-badge)             | main        | :white_check_mark: |
```

File: SECURITY.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (validcube) [reply SECURITY.md:67]: Whoops, I forgor about that.

File: SECURITY.md (hunk 1 above)
Comment (validcube) [reply SECURITY.md:69]: Sounds like a great idea, we can do that if the email is setup _(or is it already setup, if it is, lmk!)_

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+   Continuing the legacy of Vanced
+</p>
+
+# 🔒 Security Policy
+
+This document describes how to report security vulnerabilities for ReVanced API.
+
+## 🚨 Reporting a Vulnerability
+
+Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-api/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).
```
Comment (validcube) [inline SECURITY.md:67]:
```suggestion
Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-api/security/advisories/new) or reach out privately to us at security@revanced.app
```

File: SECURITY.md (hunk 1 above)
Comment (validcube) [reply SECURITY.md:69]: Resolved @ 81fd26ae55e3416f31146d58b26ac7b87659b2af

**Comment by alexandreteles** [comment]
Technically the only supported version is whatever is in production (main) and Dev works as staging grounds for changes before moving those into production.  Might want to change that before merging.

**Comment by validcube** [comment]
> Technically the only supported version is whatever is in production (main) and Dev works as staging grounds for changes before moving those into production. Might want to change that before merging.

I think development branch should be included, because it allows people to flag a security vulnerability during development which helps us fix a security hole before merging to production.

**Comment by alexandreteles** [comment]
> > Technically the only supported version is whatever is in production (main) and Dev works as staging grounds for changes before moving those into production. Might want to change that before merging.
> 
> I think development branch should be included, because it allows people to flag a security vulnerability during development which helps us fix a security hole before merging to production.

We can integrate it, but I'm not providing security updates to development versions :)

**Pull request closed without merging**